package.json

```diff
@@ -2,13 +2,17 @@
   "name": "prettierx",
   "version": "0.19.0-01-update-branch",
   "prettier-version": "2.4.0",
-  "description": "prettierx 0.19.0-01-update-branch (...) a less opinionated code formatter fork of Prettier",
+  "description": "prettierX - a less opinionated fork of Prettier code formatter",
   "bin": "./bin/prettierx.js",
   "repository": "brodybits/prettierx",
   "homepage": "https://github.com/brodybits/prettierx",
   "license": "MIT",
   "main": "./index.js",
-  "// prettierx STANDALONE TODO NOTES": "CURRENTLY NOT SUPPORTED",
+  "// prettierx package notes": [
+    "* ''standalone'' usage with browser & unpkg is currently NOT supported",
+    "  (for future consideration) - now ''commented out'' below",
+    "<!-- -->"
+  ],
```

Comment:
- [ ] I think it would be better to have fewer notes in package.json, with a pointer to more detailed notes in a Markdown file.